### PR TITLE
Files for UCD, NCCC, and Laos

### PR DESCRIPTION
**Pull request by Abby-Wheelis**
Based on agreements and other recent examples, developing configuration files for three more deployments

### CODE REVIEW

File: configs/uc-davic.nrel-op.json
Comment (shankari) [inline configs/uc-davic.nrel-op.json:9]: Let's also add the `opcode` element here so that we can eventually remove the backwards compatibility logic. See `cortezebikes` or similar for an example.
https://github.com/e-mission/nrel-openpath-deploy-configs/blob/1eaadf8a6433a6f282dff09ed055c08060b807e8/configs/cortezebikes.nrel-op.json#L9

File: configs/uc-davic.nrel-op.json
Comment (shankari) [inline configs/uc-davic.nrel-op.json:18]: I think this needs to be "California E-bike Incentive Program"?

File: configs/uc-davic.nrel-op.json
Comment (shankari) [inline configs/uc-davic.nrel-op.json:5]: The `connectUrl` should end with `/api`, as in 
https://github.com/e-mission/nrel-openpath-deploy-configs/blob/1eaadf8a6433a6f282dff09ed055c08060b807e8/configs/cortezebikes.nrel-op.json#L6

File: configs/uc-davic.nrel-op.json
Comment (shankari) [inline configs/uc-davic.nrel-op.json:28]: Ditto.

File: configs/uc-davic.nrel-op.json
Comment (shankari) [inline configs/uc-davic.nrel-op.json:57]: we want to initially include test users; we will flip the switch once they get a reasonable mass of participants

**Review by shankari — COMMENTED** [review]
Pretty good; just a few minor comments.

**Review by shankari — CHANGES_REQUESTED** [review]
@Abby-Wheelis at a high level, the name of the config should match the name of the URL.
If you look at the implementation of 
https://github.com/e-mission/nrel-openpath-join-page/blob/main/frontend/index.html#L517
and the existing join pages (e.g. https://open-access-openpath.nrel.gov/join)
we use the `study_config` primarily to support testing

In production, we infer the study name by looking at the window URL and then stripping out the first part. So if the API URL is "https://fortmorgan-openpath.nrel.gov/api/", the file should be `fortmorgan.nrel-op.json` (and the join page will eventually be https://fortmorgan-openpath.nrel.gov/join and so on

**Comment by Abby-Wheelis** [comment]
File names are now:
ca-ebike
usaid-laos-ev
fortmorgan

File: configs/ca-ebike.nrel-op.json
Comment (shankari) [inline configs/ca-ebike.nrel-op.json:1]: 😄

**Comment by Abby-Wheelis** [comment]
I've now checked on all the pages (3x2 supported languages) and all the phoneapps -- there is an issue with the Lao on the phone, even with the phone in Lao it is just showing English. Also, on the join page for the Lao, the parts specified by the config are translated but not the rest (which I'm guessing is a characteristic of the continued move to i18n and getting translations from people)

All in all, looking good to me with the exception of the Lao that still has issued to work out

**Comment by shankari** [comment]
> I've now checked on all the pages (3x2 supported languages) and all the phoneapps -- there is an issue with the Lao on the phone, even with the phone in Lao it is just showing English.

To get this to work on the phone app, you also need to change the set of languages to include `lo`
https://github.com/e-mission/e-mission-phone/blob/4a21c61f121bbf7e55a7b59c0ed82a270c6e6706/www/js/app.js#L113

and also change the translate config to pull from `master_for_platform`
(I currently do this by switching to `master_for_platform` in `locales`

```
$ git checkout --track origin/master_for_platform
```

So all subsequent pulls in `hooks/before_prepare/download_translation.js` will pull from the correct branch

Alternatively, you can wait until I merge these changes and then try using the OpenPATH app on a real phone
(The OpenPATH app is already configured as above)

> Also, on the join page for the Lao, the parts specified by the config are translated but not the rest (which I'm guessing is a characteristic of the continued move to i18n and getting translations from people)

Exactly. See the two files that I sent to Kosol for translation on June 9 (and followed up today).
The `join_page_template.docx` is essentially the template parts of the join page

**Review by shankari — APPROVED** [review]
LGTM! Don't see anything obviously wrong!

**Comment by Abby-Wheelis** [comment]
I was able to make the changes you mentioned and things showed up as expected! Still a mix for now in some places, but confirmed to work. Screenshot of a page with the Lao!

**Pull request merged**
